ustaxcourt/ef-cms-flexion
# 10228: Cypress Smoke Test (Petitioner Account Creation)

[ZenHub Card](https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/gh/flexion/ef-cms/10228)

Latest [CircleCi](https://app.circleci.com/pipelines/github/flexion/ef-cms/50124/workflows/f955b790-673d-4946-94d8-1fc4aa62f056) run. (Only running the account creation tests)

**_Tests_**
- Form Validation
- Happy Path
     - Create New Account
     - Confirm using verification link
     - Login to new Account
     - Create a Case
- Using an expired Verification Code
- Using the wrong Verification Code
